handbook/business-operations/README.md

```diff
@@ -631,6 +631,26 @@ This applies to anyone who gets paid by the hour, including consultants and hour
 
 > _**Note:** If a contributor uses their own time tracking process or tools, then it is OK to track the extra time spent tracking!  Contributors at Fleet are evaluated based on their results, not the number of hours they work._
 
+## Informing managers about hours worked
+Every Friday at 1:00pm CST, gathers hours worked for anyone who gets paid hourly by Fleet. This includes core team members and consultants, regardless of employment classification, and regardless whether inside or outside of the United States.
```

Comment:
1. CT, not CST.  (Don't want to have to wonder if it's daylight savings time or not)
2. Who gathers hours?  Ok to say "Nathan gathers" or "we gather" since it's already documented as the DRI in the rituals table.
<img width="734" alt="image" src="https://user-images.githubusercontent.com/618009/227606528-66f98a38-3b47-4f36-8ea4-951900475eec.png">